scripts/image-builder/config/blueprint_v0.0.1.toml

```diff
@@ -53,6 +53,10 @@ version = "*"
 name = "strace"
 version = "*"
 
+[[packages]]
+name = "iproute-tc"
+version = "*"
```

Comment:
Why do we need it here? I'd add it to base blueprints under test.

Reply:
What are the blueprints under test? As I can see on Jenkins jobs, this is the blueprint used to generate the image that is deployed to run RF and ginkgo tests. Am I missing something?